### ISSUE TITLE
About COCO Suponly

### ISSUE THREAD

**Issue by Luffy03** [LiheYoung/UniMatch#6]
Hello, thanks for your inspiring work!
I tried to reproduce the experiments of COCO dataset (SupOnly) with your provided code 'supvised.py'. But the results are far from your reported results. Would you please provide your logs and checkpoints for COCO (SupOnly)? 
![image](https://user-images.githubusercontent.com/52514670/193440640-34b1f084-bb0a-4c52-8ca2-742877556a26.png)

**Comment by LiheYoung** [comment]
Thank you for your interest.

Indeed we did not run the COCO SupOnly experiments by ourselves. These results are borrowed from [PC<sup>2</sup>Seg](https://arxiv.org/abs/2108.09025). You can run our semi-supervised codes on the COCO. Or you may upload your SupOnly logs first, then I will also try to run them when I am free. :)

**Comment by Luffy03** [comment]
Thanks for your kind comment. Here is my training log for COCO (SupOnly 1/512) with code 'supervised.py'. I conducted the experiment with 8 3090 GPUs, but the results are very weird.
[20221002_162900.txt](https://github.com/LiheYoung/UniMatch/files/9692380/20221002_162900.txt)

**Comment by LiheYoung** [comment]
Below are some of my explorations with our [supervised.py](https://github.com/LiheYoung/UniMatch/blob/main/supervised.py). Overall, the COCO SupOnly experiments should be trained for more epochs, due to the extremely limited data and challenging class space. I use a total batch size of ``16`` (``4`` images/gpu on ``4`` gpus), with a base learning rate ``0.004``, and ``lr_multi`` is set as ``10.0`` following the original config.
| Epochs   | 1/512  | 1/256 | 1/128 |  1/64 |  1/32 | 
| :------: | :-------: | :-------: | :-------: | :-------: | :-------: |
| 30 | 0.86 | 0.78 | 6.79 | 24.94 | 37.26 |
| 120 | 1.67 | 15.55 | 31.77 | 37.89 | 43.17 |
| 480 | 18.27 | 27.08 | 33.16 | 39.07 | 43.14 |
| |
| [PC<sup>2</sup>Seg](https://arxiv.org/abs/2108.09025) | 22.9 | 28.0 | 33.6 | 37.8 | 42.2 |

The reproduced results with more training iterations are close to that reported in PC<sup>2</sup>Seg. It is recommended to set fixed training iterations across different partitions.

As aforementioned, we directly borrow the SupOnly results from PC<sup>2</sup>Seg. To reproduce our work, you may try the semi-supervised codes of FixMatch or UniMatch. Thank you.

**Comment by Luffy03** [comment]
Thanks for your kind comment. Your explorations seem intuitive. By the way, would you please provide the results of FixMatch? It would help me a lot.

**Comment by LiheYoung** [comment]
Do you mean the FixMatch results on the COCO dataset? If so, you may refer to our Appendix A. We have included FixMatch results there.

<img width="700" alt="截屏2022-10-03 上午11 06 13" src="https://user-images.githubusercontent.com/47178369/193494781-02c9d018-c509-41e4-af02-6a150421ef1e.png">

**Comment by Luffy03** [comment]
Sorry for my neglect. Thanks for your kind answer.
